src/formatting/visitor.rs

```diff
@@ -181,18 +181,56 @@ impl<'b, 'a: 'b> FmtVisitor<'a> {
     }
 
     /// Remove spaces between the opening brace and the first statement or the inner attribute
-    /// of the block.
-    fn trim_spaces_after_opening_brace(
+    /// of the block, fast-forwarding this position of the visitor.
+    /// If `preserve_block_start_blank_lines` is true, the return value contains the newlines that
+    /// should be preserved.
+    pub(crate) fn trim_spaces_after_opening_brace(
```

Comment:
I kinda feel like we should rename this function to better represent what it's doing now. Whereas before it was indeed trimming things, it's now responsible for determining the newline story between the opening brace and first element within the block. 

What do you think?

Reply:
Good point, renamed to `advance_to_first_block_item`